### PR TITLE
Sql query printing

### PR DESCRIPTION
**Pull request by bojanserafimov**
Implement query printing such that the result can be pasted into a database console and ran.

### CODE REVIEW

File: graphql_compiler/compiler/sqlalchemy_extensions.py [hunk 1]
```diff
@@ -40,3 +40,26 @@ def not_contains_operator(collection, element):
...
+    """Return a string form of the parameterized query.
+
+    Args:
+        query: sqlalchemy.sql.selectable.Select
+        dialect: sqlalchemy.engine.interfaces.Dialect
+
+    Returns:
+        string that can be ran using sqlalchemy.sql.text(result)
+    """
+    class LiteralCompiler(sqlalchemy.dialects.mssql.base.MSSQLCompiler):
```
Comment (pmantica1) [inline graphql_compiler/compiler/sqlalchemy_extensions.py]: Can you either make sure this works for dialects other than mssql, raise an assertion if this is not a mssql dialect,  or change the docstring of this function?

File: graphql_compiler/tests/test_sqlalchemy_extensions.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    def test_print_query_mssql_list_argument(self):
+        animal = self.sql_schema_info.vertex_name_to_table['Animal'].alias()
+        query = sqlalchemy.select([animal.c.name]).where(
+            animal.c.name.in_(sqlalchemy.bindparam('names', expanding=True))
+        )
+        text = print_sqlalchemy_query_string(query, mssql.dialect())
+        expected_text = '''
+             SELECT [Animal_1].name
+             FROM db_1.schema_1.[Animal] AS [Animal_1]
+             WHERE [Animal_1].name IN :names
```
Comment (pmantica1) [inline graphql_compiler/tests/test_sqlalchemy_extensions.py]: Have you tried seeing if you could fill in the parameter of this query by using pymssql or pydobc and actually running it?

Comment (bojanserafimov) [reply graphql_compiler/tests/test_sqlalchemy_extensions.py]: yes

File: graphql_compiler/compiler/sqlalchemy_extensions.py (hunk 1 above)
Comment (bojanserafimov) [reply graphql_compiler/compiler/sqlalchemy_extensions.py]: Made it dialect-agnostic

**Pull request merged**